shell/platform/android/io/flutter/plugin/editing/InputConnectionAdaptor.java

```diff
@@ -262,6 +278,16 @@ public boolean setSelection(int start, int end) {
     return result;
   }
 
+  // TODO(justinmc): Scribe stylus gestures should be supported here.
+  // https://github.com/flutter/flutter/issues/156018
+  @TargetApi(API_LEVELS.API_34)
+  @RequiresApi(API_LEVELS.API_34)
+  @Override
+  public void performHandwritingGesture(
+      HandwritingGesture gesture, Executor executor, IntConsumer consumer) {
+    executor.execute(() -> consumer.accept(HANDWRITING_GESTURE_RESULT_UNSUPPORTED));
```

Comment:
My intention was to indicate that no HandwritingGesture is currently supported. Later I would add support in this method (see https://github.com/flutter/engine/pull/55568).

However I think this doesn't matter because I already didn't register support when I enable stylus handwriting. I'll just remove this method for now until that follow-up PR.